packages/ckeditor5-dev-ci/bin/notifycirclestatus.js

```diff
@@ -0,0 +1,145 @@
+#!/usr/bin/env node
+
+/**
+ * @license Copyright (c) 2003-2023, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+/* eslint-env node */
+
+const allowedBranches = require( '../lib/data/allowedBranches.json' );
+const formatMessage = require( '../lib/formatMessage' );
+const printLog = require( '../lib/printLog' );
+const slackNotify = require( 'slack-notify' );
+
+const ALLOWED_EVENTS = [
+	'api',
+	'webhook'
+];
+
+// This script assumes that is being executed on Circle CI.
+// Step it is used on should have set value: `when: on_fail`, since it does not
+// check whether the build failed when determining whether the notification should be sent.
+// Described environment variables starting with "CKE5" should be added by the integrator.
+
+const {
+	/**
+	 * @enum {String} Token to a Github account with the scope: "repos". It is required for obtaining an author of
```

Comment:
```suggestion
	 * Token to a Github account with the scope: "repos". It is required for obtaining an author of
```

I think we can skip the `@enum`/`@type` annotations as each variable extracted from `process.env` will be a string.